frontend/lib/resolution/prims.cpp

```diff
@@ -72,10 +72,10 @@ static bool toParamIntActual(const QualifiedType& type, int64_t& into) {
   return false;
 }
 
-static bool toParamStringActual(const QualifiedType& type, UniqueString& into) {
+static bool paramStringBytesHelper(const QualifiedType& type, UniqueString& into, bool isString) {
   if (type.kind() == QualifiedType::PARAM) {
     if (auto t = type.type()) {
-      if (t->isStringType()) {
+      if (t->isBytesType() || (t->isStringType() && isString)) {
```

Comment:
```suggestion
      if ((t->isBytesType() && !isString) || (t->isStringType() && isString)) {
```
?

Or can string be interpreted as bytes, and this is on purpose?

Reply:
nah, I just didn't expect it to be called that way. I added it as a check.